test/server/lib/route-helpers.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const moment = require('moment');
+const timeshift = require('timeshift');
+const stub = sinon.stub().resolves('finished');
+const { dateChecker, promiseResponder, dashboard, home } = require('proxyquire')('../../../src/lib/route-helpers', {
+	'./model': stub
+});
+
+describe('lib/routesHelpers', () => {
+	describe('dateChecker', () => {
+			describe('checks for invalid dates', () => {
+				it('should return false when the input is not a valid date', () =>
+					expect(dateChecker('2019-09-50'))
+						.to.equal('Invalid date'));
```

Comment:
This test doesn't match the description.

Reply:
fixed